com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -1526,6 +1526,34 @@ void RenderGizmos(RenderGraph renderGraph, HDCamera hdCamera, TextureHandle colo
             return executed;
         }
 
+        private class UpdatePostProcessScreenSizePassData
+        {
+            public int postProcessWidth;
+            public int postProcessHeight;
+            public HDCamera hdCamera;
+            public ShaderVariablesGlobal shaderVariablesGlobal;
+        }
+
+        internal void UpdatePostProcessScreenSize(RenderGraph renderGraph, HDCamera hdCamera, int postProcessWidth, int postProcessHeight)
+        {
+            using (var builder = renderGraph.AddRenderPass("Update RT Handle Scales CB", out UpdatePostProcessScreenSizePassData passData))
+            {
+                passData.hdCamera = hdCamera;
+                passData.shaderVariablesGlobal = m_ShaderVariablesGlobalCB;
+                passData.postProcessWidth  = postProcessWidth;
+                passData.postProcessHeight = postProcessHeight;
+                builder.AllowPassCulling(false);
```

Comment:
No need for this line

Reply:
Done. May I ask then, when are render passes culled? Since this pass is not using any render targets (are render passes withouth any targets automatically not culled?)

Reply:
This indeed feels wrong on second read (was ealry morning first time around :D ) Though I see most of the passes we do that just push constants don't have that set. 

Paging in rendergraph owner @JulienIgnace-Unity

Reply:
@julienb-unity I still dont know, but I do think I should leave the AllowPassCulling here right?

Reply:
Wrong @JulienIgnace-Unity  :D

Reply:
@kecho I think you referenced the wrong Julien... 😃

Reply:
LOL!! sorry man!